packages/big-design/src/components/Accordion/spec.tsx

```diff
@@ -0,0 +1,65 @@
+import { fireEvent, render } from '@test/utils';
```

Comment:
Instead of all the snapshot tests, can you please render an accordion with content and then do something like `.toBeInTheDocument()`?

Reply:
Thanks, changed to version with `.toBeInTheDocument()`.